cli/options.go

```diff
@@ -333,9 +333,9 @@ func ProjectFromOptions(options *ProjectOptions) (*types.Project, error) {
 		} else if nameFromEnv, ok := options.Environment[ComposeProjectName]; ok && nameFromEnv != "" {
 			opts.Name = nameFromEnv
 		} else {
-			opts.Name = filepath.Base(absWorkingDir)
+			opts.Name = strings.ToLower(filepath.Base(absWorkingDir))
```

Comment:
What if the name comes from the environment?
I think the `strings.ToLower` should be applied inside `normalizeName` so that it doesn't depend from where it comes.

Reply:
right. I prefer less magic with a name I've set explicitly, but let's go this way